packages/cubejs-api-gateway/src/gateway.ts

```diff
@@ -176,6 +178,10 @@ export interface ApiGatewayOptions {
 export class ApiGateway {
   protected readonly refreshScheduler: any;
 
+  protected readonly scheduledRefreshContexts: any;
```

Comment:
`ApiGatewayOptions['scheduledRefreshContexts']` instead of `any` will be better, similar for `scheduledRefreshTimeZones`